docs/cards.md

```diff
@@ -138,19 +138,34 @@ Try hovering over then clicking on the cards below:
 
 :::{card} Clickable Card (external)
 :link: https://example.com
-:link-alt: example
+:link-alt: example.com
 
 The entire card can be clicked to navigate to <https://example.com>.
+
+For **external** link cards, if you do not provide `link-alt`, the URL will be
```

Comment:
I think it would be good to add tests for this behavior, i.e.:

1. when card link is external and no link-alt is provided, use the URL provided by the link option
2. when card link is external and link-alt is provided, use the link-alt text
3. when card link is internal and no link-alt is provided, use the (section) name of the reference
4. when card link is internal and link-alt is provided, use the link-alt text
5. in all cases, apply the hide-link class

The current snapshots in this PR implicitly cover 2 and 3 but it would be good to have individual test lines that spell these conditions out. 

I'm just not entirely sure how to go about it, since all of the current tests are regression snapshot tests.

I suppose I could go ahead and add snippets for conditions 1 (card-link-external-no-link-alt) and 4 (card-link-internal-with-link-alt), which would also imply 5.

Anyway, let me know what you would like me to do. Happy to add tests in whatever way seems best.